### PR TITLE
New logging configuration choice - Video Custom Textures

### PR DESCRIPTION
**Pull request by ZexaronS**
More logging will be added here in future. Should enable less clutter in the console because other video events are not necessary for my future development around custom textures.

### CODE REVIEW

File: Source/Core/Common/Logging/LogManager.cpp [hunk 1]
```diff
@@ -87,6 +87,7 @@ LogManager::LogManager()
   m_Log[LogTypes::SP1] = new LogContainer("SP1", "Serial Port 1");
   m_Log[LogTypes::VIDEO] = new LogContainer("Video", "Video Backend");
   m_Log[LogTypes::VIDEOINTERFACE] = new LogContainer("VI", "Video Interface (VI)");
+  m_Log[LogTypes::VIDEOCUSTOMTEX] = new LogContainer("VCT", "Video Custom Textures (VCT)");
```
Comment (JosJuice) [inline Source/Core/Common/Logging/LogManager.cpp]: For log category names, we only add acronyms in parentheses if the acronym is likely to be easier to recognize than the full name. That's definitely not the case for the acronym you're adding - I've never seen it be used even a single time before. Please remove it.

Comment (leoetlino) [inline Source/Core/Common/Logging/LogManager.cpp]: And VCT for the log tag is a really non obvious name. This should likely be VIDEOCUSTOMTEX or VideoCustomTextures (not sure what the convention for those is?)

Comment (ZexaronS) [inline Source/Core/Common/Logging/LogManager.cpp]: I have asked for consultations with nobody responding I simply used the strings according to the existing pattern, so the initial changes shouldn't be surprising imo, for someone unfamiliar with the project.

Comment (JosJuice) [inline Source/Core/Common/Logging/LogManager.cpp]: Dropping video sounds fine. Just make sure to drop the acronym too.

Comment (ZexaronS) [inline Source/Core/Common/Logging/LogManager.cpp]: I agree that VCT is not obvious moniker, I realize now it's the tag used in practise, in the console output. 

Is anyone in favor of dropping the "video" moniker ?

Comment (JosJuice) [inline Source/Core/Common/Logging/LogManager.cpp]: Dropping video from this name sounds fine. Just make sure to drop the acronym too.

**Comment by ZexaronS** [comment]
I plan to add more logging entries in the Prefetcher calculation for reporting the size and time of the loaded textures, as well as the components of the system that will stop prefetching if it detects low RAM environment, where it seems fit.

File: Source/Core/Common/Logging/LogManager.cpp (hunk 1 above)
Comment (ZexaronS) [inline Source/Core/Common/Logging/LogManager.cpp]: There is inconsistency in character case already present in tags, so it makes it a bit harder to pick objectively.

**Comment by ZexaronS** [comment]
I prefer longer names in lowercase. Just now because of the removal of video moniker, I moved the thing to be alphabetically correct order, so up those there's mostly upper case ones, so it may appear wrong, but you have "FileMon" .. "Wiimote" ... "Audio" "Video" ...

**Comment by ZexaronS** [comment]
Actually, this is meant to be the Prefetcher overhaul, with the added thing of
I messed up again, I should have do this, if doing it separately, the last, not the first in order. 
I think the more practical way to do this is with multiple commits, not multiple PRs. 

Is anyone protesting that I rename this PR into "Custom Textures Prefetcher Overhaul" or something similar, I have already stated above what the plan is in detail.

**Pull request closed without merging**